community/chromium/build

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh -e
+
+kiss l musl && {
```

Comment:
mind using full form `l` -> `list` ?

Reply:
yeah sure, need to fix up the packaging part of the build aswell